.gitignore

```diff
@@ -68,6 +68,9 @@ yarn-debug.log*
 !/storage/.keep
 /public/uploads
 
-# Ignore master key for decrypting credentials and more.
+# Ignore master key for decrypting credentials and more as well as database config file.
 /config/master.key
 /config/database.yml
```

Comment:
So, I noticed you ignored the `database.yml` file, and it would be great if you could commit it to the project, so that I can check that the database adapter is using Pstgresql:

![image](https://github.com/kkolade/rails_blog_app/assets/93520178/ec453e31-0789-482c-ab48-ddb20d1024d6)